### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by nex693** [f0Re3t/nex-ac_rakcheat#13]
these 2 tricks work
can you block

1 : https://www.blast.hk/threads/32887/

2 : https://www.blast.hk/threads/34601/

**Comment by f0Re3t** [comment]
Hi, I'll see how the time goes

**Comment by nex693** [comment]
Thanks. I would appreciate if you report these working cheats to Nexius

**Comment by NexiusTailer** [comment]
Seems like these are rather old releases that I have already checked and they cannot be fully fixed (as far as I remember, they only work from some limited very close distances or something like that).

**Comment by nex693** [comment]
There are servers blocking

**Comment by nex693** [comment]
@NexiusTailer

**Comment by NexiusTailer** [comment]
Which servers?

**Comment by nex693** [comment]
I can't give it here. Do you have Discord?

**Comment by NexiusTailer** [comment]
No

**Comment by nex693** [comment]
Anti Troll + teleport should be developed

**Comment by nex693** [comment]
https://www.blast.hk/threads/29921/

Buddha is working

**Comment by NexiusTailer** [comment]
It is already developed

**Comment by nex693** [comment]
believe me it has not been developed enough

**Comment by nex693** [comment]
https://www.blast.hk/threads/29921/

Buddha is working

**Comment by NexiusTailer** [comment]
No, this is fully developed from 2015

**Comment by nex693** [comment]
You never know before you try it, this cheat works / gg (id)
/pizda
/cslap
(you should be in the vehicle)

**Comment by nex693** [comment]
Additionally, they beam from a long distance on my server so it's undeveloped.

**Comment by NexiusTailer** [comment]
> You never know before you try it, this cheat works / gg (id)
> /pizda
> /cslap
> (you should be in the vehicle)

No, it was tested right after its release

**Comment by nex693** [comment]
They run it on the server. Can you give me a solution?

**Comment by NexiusTailer** [comment]
Probably no, as I wrote above

**Comment by nex693** [comment]
Can't solve, f0Re3t can only solve

**Comment by NexiusTailer** [comment]
I don’t think so. Each new cheat is tested by us almost immediately after its release, and if, as a result, it has not been fixed (or continues to work from some certain distances and speeds), then there is no normal fix for it. But I told you about this several times already, so I see no reason to repeat myself.

**Comment by nex693** [comment]
no fix but you're Nexius, you need to fix it
Do the impossible, you're Nexius
there is no impossible for you!
you should do, you are the best

Nexius MAKES IMPOSSIBLE!


50 users are waiting for big updates from you
(Server user)

**Comment by nex693** [comment]
You are Nexius there is no such thing as impossible

**Comment by nex693** [comment]
new bug about anti cheat; When you come back afk, you get kicked 

(code: 003)
(try on vehicle)

@NexiusTailer

**Comment by nex693** [comment]
this error was not present in the old version

**Comment by nex693** [comment]
new mistake: sultan kicks when changing vehicle (modified)

[13/04/2021 12:17:09] [Nex-AC debug] Money: 13190133, old money: 13190133, component price: 1600
[13/04/2021 12:17:09] [Nex-AC]  ID : 1. code: 023 (3)


@NexiusTailer

**Comment by f0Re3t** [comment]
> new mistake: sultan kicks when changing vehicle (modified)
> 
> [13/04/2021 12:17:09] [Nex-AC debug] Money: 13190133, old money: 13190133, component price: 1600
> [13/04/2021 12:17:09] [Nex-AC] ID : 1. code: 023 (3)
> 
> @NexiusTailer

maybe you are crooked or not correctly configured anti-cheat?

**Comment by nex693** [comment]
> > new mistake: sultan kicks when changing vehicle (modified)
> > [13/04/2021 12:17:09] [Nex-AC debug] Money: 13190133, old money: 13190133, component price: 1600
> > [13/04/2021 12:17:09] [Nex-AC] ID : 1. code: 023 (3)
> > @NexiusTailer
> 
> maybe you are crooked or not correctly configured anti-cheat?

case 23: return 1;
I solved it by doing. Is it correct?

**Comment by f0Re3t** [comment]
> > > new mistake: sultan kicks when changing vehicle (modified)
> > > [13/04/2021 12:17:09] [Nex-AC debug] Money: 13190133, old money: 13190133, component price: 1600
> > > [13/04/2021 12:17:09] [Nex-AC] ID : 1. code: 023 (3)
> > > @NexiusTailer
> > 
> > 
> > maybe you are crooked or not correctly configured anti-cheat?
> 
> case 23: return 1;
> I solved it by doing. Is it correct?

can you give me a sample code to trigger this on a pure game mod?
